src/record/export/types/field.ts

```diff
@@ -48,7 +48,6 @@ export type OneOf =
   | KintoneRecordField.Modifier
   | KintoneRecordField.UpdatedTime
   | KintoneRecordField.SingleLineText
-  // eslint-disable-next-line @typescript-eslint/ban-types
   | KintoneRecordField.Number // Although ESLint recognizes it as primitive type, this type is defined above in this file.
```

Comment:
NITS: `Although ESLint recognizes...`のコメントも消して良さそう

```suggestion
  | KintoneRecordField.Number
```

Reply:
[6738f76](https://github.com/kintone/cli-kintone/pull/730/commits/6738f7659bb7f7fa947d863a44522dd37847cf39) で対応